chirp/drivers/tdh8.py

```diff
@@ -906,7 +893,47 @@ def get_memory(self, number):
         elif int(scan_val_list[number - 1]) == 1:
             mem.skip = ''
 
-        mem.mode = _mem.wide and "NFM" or "FM"
+        if self._is_txinh(_mem):
+                mem.duplex = ""
```

Comment:
CHIRP uses `duplex="off"` to mean "TX inhibit. This `duplex=""` means "same TX and RX frequency.